tf2onnx/tfonnx.py

```diff
@@ -1664,21 +1664,35 @@ def floormod_op(ctx, node, name, args):
     return nodes
 
 
-def all_op(ctx, node, name, args):
+def reduce_logic_op(ctx, node, name, args):
     # T output = All(T x, list(int) reduce_indices, @bool keepdims)
+    # T output = Any(T x, list(int) reduce_indices, @bool keepdims)
     reduce_dim = node.inputs[1].get_tensor_value()
     if isinstance(reduce_dim, np.ndarray):
         reduce_dim = reduce_dim.tolist()
     else:
         reduce_dim = [reduce_dim[0]]
+    for i in reduce_dim:
```

Comment:
use all() for a check

or you can use node.input[0]'s shape length to calculate the negative axis's corresponding positive axis number.

Reply:
done